app/src/main/java/tftpserver/handlers/GetHandler.java

```diff
@@ -1,8 +1,57 @@
 package tftpserver.handlers;
 
+import java.io.IOException;
+import java.net.DatagramPacket;
+import java.net.DatagramSocket;
+import java.net.SocketAddress;
+import java.net.SocketException;
+import java.util.List;
+
+import tftpserver.PacketHelper;
+import tftpserver.db.InMemDb;
+
 public class GetHandler implements DataTransferHandler {
+    private DatagramSocket sock = null;
+    private SocketAddress clientAddress = null;
+    private String filename = null;
+    private short errorCode=1;
+    private short ACK_PACKET_LEN=4;
+    byte arr[] = new byte[1024];
```

Comment:
Shouldn't the size of this byte array be `ACK_PACKET_LEN`? Why do you need a buffer of 1kb to hold an ack packet?